### PR TITLE
Add execution of freshclam after enable antivirus

### PR DESCRIPTION
**Pull request by papajulio**
The cron could take up to one hour to execute the
antivirus update and make fail antivirus till then

### CODE REVIEW

File: main/antivirus/stubs/freshclam.profile.mas [hunk 1]
```diff
@@ -2,6 +2,7 @@
   $observerScript
 </%args>
 /run/clamav/clamd.ctl rw,
+/proc/*/status r,
```
Comment (sixstone-qq) [inline main/antivirus/stubs/freshclam.profile.mas]: Why is this change required?

Comment (papajulio) [inline main/antivirus/stubs/freshclam.profile.mas]: To avoid an AppArmor error that appears after the execution of freshclam.

Comment (sixstone-qq) [inline main/antivirus/stubs/freshclam.profile.mas]: Could you add this change to the ChangeLog? It is relevant for security purposes.

Comment (papajulio) [inline main/antivirus/stubs/freshclam.profile.mas]: Added https://github.com/Zentyal/zentyal/commit/378df195971f62c193c47c9c91093ab13639f081 sorry for the rush but this was breaking several suites in the tests.
